.github/workflows/ci.yml

```diff
@@ -28,11 +28,13 @@ jobs:
       fail-fast: false
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v2
-      - name: Prepare Simulator Runtimes
-        run: Scripts/github/prepare-simulators.sh ${{ matrix.platform }}
```

Comment:
good to see this go away! Haven't needed this for a few release cycles now. We should probably delete the script from git as well though.

Reply:
I think we needed this because the iOS versions trailed behind a bit. Agreed we should update to the snapshots to latest iOS version too though.

Reply:
I'll push an update to delete the script, afaik if we pass the destination to Xcode build it should prepare the right sim